### ISSUE TITLE
Since Spring Cleaning Update, dota won't start on DISPLAY=:0.1

### ISSUE THREAD

**Issue by mjml** [ValveSoftware/Dota-2#811]
Description: I have two monitors set up as two different displays, :0.0 and :0.1. 

Before the Spring Cleaning patch, I used to run the steam client on :0.1 and the Dota window would open on this display. Right after updating to the Spring Cleaning path, I tried to run dota from that display and got a steam client message window popup dialog titled, "Unable to start game". The message reads, 

"Failed to create OpenGL context. 
Your graphics card must support at least OpenGL v3.1."

Clearly this requirement was not introduced by Spring Cleaning, and my gpu (NVidia GTX 590) does support this OpenGL version (in fact up to v4.5.0). Just to be sure, I downloaded the latest NVidia driver (361.28), installed it and tried again. The same thing happened.

I found that you could launch dota 2 from the command line using the steam client run.sh program which sets up some environment variables and then launches dota2.sh. This actually worked just fine although it started the program on display :0.0. Figuring that the :0.1 display was the problem, I started the steam client on :0.0 and was able to run dota 2 on the :0.0 just fine.

(But I still hope that you'll fix this to allow me to run on :0.1 once again.)

**Comment by Tele42** [comment]
@mjml nvidia 361.28 has a known defective version of libglvnd bundled with it which causes this message, it only affects 361.28. Also, there is a known workaround by adding `__GLVND_DISALLOW_PATCHING=1 %command%` to the launch options of affected games.

**Comment by mjml** [comment]
Yes although I actually installed this version of the driver in response to the problem, which also occurred with the previous version of the driver.

**Comment by gdrewb-valve** [comment]
Do things work on 361.28 if you use the workaround?

**Comment by mjml** [comment]
Yes, using 361.28 I currently have dota running on :0.0 and it works fine. I still get the message if I try to run on :0.1.

**Comment by gdrewb-valve** [comment]
Do other OpenGL programs work and start on the correct screen?

**Comment by mjml** [comment]
Yes, both glxgears and glxinfo start on :0.1. I put the glxinfo output into a pastebin if you're interested. The OpenGL version string is "4.5.0 NVIDIA 361.28".

http://pastebin.ca/3413466

..[EDIT] It seems the pastebin has been corrupted. I will look for another place to paste it.
